src/http/api/views/account.follows.view.ts

```diff
@@ -141,10 +183,15 @@ export class AccountFollowsView {
             return error('not-an-actor');
         }
 
+        const followeeAccount = await this.db('accounts')
+            .where('ap_id', actor.id?.toString() || '')
```

Comment:
This should be using `ap_id_hash` because `ap_id` is not indexed - there's an example here https://github.com/TryGhost/ActivityPub/blob/de2e070f31c860a027c5d9f22dfaf68dbe70fceb/src/post/post.repository.knex.ts#L154-L156

Reply:
Done in https://github.com/TryGhost/ActivityPub/pull/557/commits/92b298171accb7c5f0b0ab5cc240b5ddff22d18c